### PR TITLE
Prepare v0.12.2

### PR DESCRIPTION
**Pull request by jxs**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 0.13.2
+
+- Bound `Active`'s `pending_frames` to enforce backpressure. 
+  See [460baf2ccb7d5982b266cb3cb9c0bdf75b4fb779](https://github.com/libp2p/rust-yamux/commit/460baf2ccb7d5982b266cb3cb9c0bdf75b4fb779)
```
Comment (mxinden) [inline CHANGELOG.md:4]:
```suggestion
  See [460baf2](https://github.com/libp2p/rust-yamux/commit/460baf2ccb7d5982b266cb3cb9c0bdf75b4fb779)
```

I believe the short commit hash suffices here.

**Review by mxinden — APPROVED** [review]
Thanks for tackling this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 0.13.2
+
+- Bound `Active`'s `pending_frames` to enforce backpressure. 
+  See [460baf2](https://github.com/libp2p/rust-yamux/commit/460baf2ccb7d5982b266cb3cb9c0bdf75b4fb779)
```
Comment (jxs) [inline CHANGELOG.md:4]:
```suggestion
  See [460baf2](https://github.com/libp2p/rust-yamux/commit/460baf2ccb7d5982b266cb3cb9c0bdf75b4fb779)
  
```

**Comment by dhuseby** [comment]
Thanks for the review @mxinden

**Pull request merged**